python/semantic_kernel/functions/kernel_function_from_prompt.py

```diff
@@ -6,7 +6,7 @@
 from html import unescape
 from typing import TYPE_CHECKING, Any
 
-import yaml
+import yaml  # type: ignore
```

Comment:
why is this needed?

Reply:
Mypy throws an error

Reply:
<img width="1066" alt="image" src="https://github.com/user-attachments/assets/2ef86b99-e241-4a5d-9afe-6d2e56b305a3" />

Reply:
Seems related: https://github.com/python/mypy/issues/10632